test/setup/anInitialSetUp.e2e.ts

```diff
@@ -36,14 +36,13 @@ describe('An Initial SetUp', async () => {
     // create and write in a text file
     fs.writeFileSync(authFilePath, sfdxAuthUrl);
 
-    // For sfdx -> sf, remove the two lines below this comment block and uncomment the following line instead
-    // await exec(`sf org login sfdx-url --sfdx-url-file ${authFilePath} --set-default --alias ${devHubAliasName}`);
-    const authorizeOrg = await exec(`sfdx org:login:sfdx-url -d -f ${authFilePath}`);
+    const authorizeOrg = await exec(`sf org login sfdx-url -d -f ${authFilePath}`);
+    // const authorizeOrg = await exec(`sfdx org:login:sfdx-url -d -f ${authFilePath}`);
```

Comment:
Why did you take out the colons here?  Can you put them back and remove the commented code in Line 40?